signer/src/block_observer.rs

```diff
@@ -791,7 +791,12 @@ mod tests {
             .with_stacks_client(test_harness.clone())
             .with_emily_client(test_harness.clone())
             .with_bitcoin_client(test_harness.clone())
-            .modify_settings(|settings| settings.signer.sbtc_bitcoin_start_height = min_height)
+            .modify_settings(|settings| {
+                settings.signer.sbtc_bitcoin_start_height = match min_height {
```

Comment:
You could probably use a simple `.map()` instead of a whole `match` (here and below).

Reply:
[0808b24](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/0808b247642e64f0e2a191460ed1ad94ecfde6bd)